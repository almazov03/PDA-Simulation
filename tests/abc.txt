STARTING_NON_TERMINAL=$S$
$S$ = { $S$ "a" }
$S$ = { $S$ "b" }
$S$ = { $S$ "c" }
$S$ = { EPS }