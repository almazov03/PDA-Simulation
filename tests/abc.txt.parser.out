STARTING_NON_TERMINAL=$START$
$START$ = { EPS }
$S$ = { "a" }
$S$ = { "b" }
$S$ = { "c" }
$S$ = { "a" $S'$ }
$S$ = { "b" $S'$ }
$S$ = { "c" $S'$ }
$S'$ = { "a" $S'$ }
$S'$ = { "a" }
$S'$ = { "b" $S'$ }
$S'$ = { "b" }
$S'$ = { "c" $S'$ }
$S'$ = { "c" }
$START$ = { "a" }
$START$ = { "b" }
$START$ = { "c" }
$START$ = { "a" $S'$ }
$START$ = { "b" $S'$ }
$START$ = { "c" $S'$ }

